paf-mvp-client-express/src/client-node.ts

```diff
@@ -392,6 +406,28 @@ export class ClientNode extends Node {
     }
   };
 
+  verifyTransmission = async (req: Request, res: Response, next: NextFunction) => {
+    const message = fromDataToObject<TransmissionResultSignatureData>(req.body);
+    try {
+      const verificationResult = await this.client.verifyTransmissionResult(message);
+      if (!verificationResult.isValid) {
+        const error: NodeError = {
+          type:
+            verificationResult.errors[0] instanceof UnableToIdentifySignerError
+              ? 'UNKNOWN_SIGNER'
+              : 'VERIFICATION_FAILED',
+          details: verificationResult.errors[0].message,
+        };
+        next(error);
+      } else {
+        res.json(); // Empty response is fine if code is 200
```

Comment:
We can maybe use 204 Http code here ?

Reply:
Good point.
Updated for this endpoint and `verifySeed`